rero_ils/modules/patrons/api.py

```diff
@@ -825,6 +825,17 @@ def basic_query(channel):
             process_bulk_queue.apply_async()
 
 
+    def get_age(self):
```

Comment:
This method could be a `@property` of patron.
```
@property
def age(self):
    ...
```

Reply:
Ok